api/alembic/versions/b99dce009b9f_change_default_fuel_types_for_coastal_.py

```diff
@@ -12,7 +12,7 @@
 
 # revision identifiers, used by Alembic.
 revision = 'b99dce009b9f'
-down_revision = '52c8a568364e'
+down_revision = '35acf3b96d8a'
```

Comment:
Isn't changing the down revision on something that's already been pushed to production going to cause alembic to fail when we go to production?